packages/services/NetworkService/index.ts

```diff
@@ -164,6 +134,62 @@ class NetworkService {
         })
         .catch((err: any) => reject(err));
     });
+  };
```

Comment:
Why ?

Reply:
I think more than 3 lines of code writing inline is not human readable.